examples/both/flake.nix

```
@@ -59,7 +60,8 @@
           # Configurations for macOS machines
           darwinConfigurations = {
             # TODO: Change hostname from "example1" to something else.
-            example1 = self.nixos-flake.lib.mkMacosSystem "aarch64-darwin" {
+            example1 = self.nixos-flake.lib.mkMacosSystem {
+              nixpkgs.system = "aarch64-darwin";
```

```suggestion
              nixpkgs.hostPlatform = "aarch64-darwin";
```

These options were recently ported to nix-darwin too. I don't think it generates a `hardware-configuration.nix`.

---

Similar changes apply for the files below.